packages/react-server/src/ReactFlightServer.js

```diff
@@ -2152,7 +2152,11 @@ function visitAsyncNode(
                 owner: node.owner,
                 stack: filterStackTrace(request, node.stack),
               });
-              markOperationEndTime(request, task, endTime);
+              // Mark the end time of the await. If we're aborting then we don't emit this
+              // to signal that this never resolved inside this render.
+              if (request.status !== ABORTING) {
+                markOperationEndTime(request, task, endTime);
+              }
```

Comment:
Does that not cause an infinite span in the performance track?

Reply:
No because we skip writing it if there's no end time.

https://github.com/facebook/react/blob/7a3ffef70339c10f8d65a27b88cd73bfbe13eb8a/packages/react-client/src/ReactFlightClient.js#L3213

However, I do want to add some kind of marker in the track to log a halted entry.

Reply:
Maybe it would be useful to have some kind of end marker to know what time it was aborted though.

Reply:
https://github.com/facebook/react/pull/33641